overview/index.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: index
+title: Overview
+permalink: /overview/
+---
+
+This page provides an overview of how Kroxylicious works.  For more details, please refer to the [documentation](./kroxylicious).
+
+#### What is Kroxylicious?
+
+Kroxylicious is an Apache Kafka&#174; protocol-aware proxy.  It can be used to layer uniform behaviours onto a
+Kafka based system in areas such as data-governance, security, policy enforcement and audit without needing to
+change either the applications or the Kafka Cluster.
+
+##### Filters
```

Comment:
I'm not quite sure where it goes, but I think we also want to mention that we just pass the bytes straight through if no filters are interested in the particular API key.

Reply:
I think that might be deeper than I'm intending.  The overview intended audience is the Decision Maker.  I want to give them sufficient context to understand the use-case (/avoid the use-cases having to repeat core stuff)

Reply:
I understand the hesitation but I think one of Kroxylicious important properties is the minimal costs for messages which are not being manipulated.